cfgov/unprocessed/js/routes/browse-basic/index.js

```diff
@@ -0,0 +1,26 @@
+/* ==========================================================================
+   Scripts for /browse-basic/.
+   ========================================================================== */
+
+'use strict';
+
+// Required polyfills for <IE8.
+require( '../../modules/polyfill/query-selector' );
+
+// List of organisms used.
+var Expandable = require( '../../molecules/Expandable' );
+var ExpandableGroup = require( '../../organisms/ExpandableGroup' );
+
+var expandables = document.querySelectorAll( '.expandable-prototypes .m-expandable' );
```

Comment:
Why nest in `.expandable-prototypes` and not just target `.m-expandable` directly? Same question for groups on L:21